qoe-metrics-dashboard/src/app/types/qoe-report.type.ts

```diff
@@ -0,0 +1,137 @@
+export interface ReceptionReport {
+  contentURI: string;
+  clientID?: string;
+  qoeReports?: QoeReport[]; // Zero or more QoE reports
+}
+
+export interface QoeReport {
+  periodID: string;
+  reportTime: Date; // Assuming you want to use JavaScript Date objects
+  reportPeriod: number;
+  qoeMetrics: QoeMetric[]; // One or more QoE metrics
+}
+
+export type QoeMetric = HttpList | RepSwitchList | AvgThroughput[] | number | BufferLevel | PlayList | MpdInformation[] ;
+
+export interface HttpList {
+  httpListEntries?: HttpListEntry[];
+}
+
+export interface HttpListEntry {
+  tcpid?: number;
+  type?: ExtensibleHttpEntryResourceType;
+  url: string;
+  actualUrl?: string;
+  range?: string;
+  trequest: Date;
+  tresponse: Date;
+  responsecode?: number;
+  interval?: number;
+  traces?: HttpThroughputTrace[];
+}
+
+export interface HttpThroughputTrace {
+  s: Date;
+  d: number;
+  b: number[]; 
+}
+
+export interface RepSwitchList {
+  repSwitchEvents?: RepSwitchEvent[];
+}
+
+export interface RepSwitchEvent {
+  to: string;
+  mt?: number;
+  t?: Date;
+}
+
+export interface AvgThroughput {
```

Comment:
Das können wir glaube ich auch erstmal rauslassen. Er meinte ja, das mehr als die Metrics die in sample.xml enthalten sind im Moment überhaupt nicht generiert werden